app/views/exam_templates/_split_pdf_log.html.erb

```diff
@@ -0,0 +1,28 @@
+<div>
+  <% current_incomplete_num_groups = split_pdf_log.num_groups_in_incomplete %>
+  <% if current_incomplete_num_groups == previous_incomplete_num_groups && split_pdf_log.num_pages_qr_scan_error == 0 %>
+    <p class="log-green-message">
+      <%= "#{t('exam_templates.split_pdf_log.log')} ##{i}" %> - <%= split_pdf_log.uploaded_when.strftime("%d %b %y at %I:%M:%S %p") %>
```

Comment:
`split_pdf_log.uploaded_when` shows a date format like this `2017-06-23 16:56:58 -0400`. I thought 0400 at the end was unnecessary, so this time I changed it slightly to make it look like this `2017-06-23 09:56:58 AM`. But please let me know if I should keep it as it is.